scripts/combine_lightcurves.py

```diff
@@ -4,7 +4,7 @@
 import shutil
 import json
 import argparse
-from json.decoder import JSONDecodeError
+#from json.decoder import JSONDecodeError
```

Comment:
Can you just remove this commented line and then it looks good to merge. I like that little flask listener, that's snazzy.